wallet/src/de/schildbach/wallet/ui/dashpay/CreateIdentityService.kt

```diff
@@ -322,7 +322,13 @@ class CreateIdentityService : LifecycleService() {
         val emptyProfile = DashPayProfile(blockchainIdentity.uniqueIdString, blockchainIdentity.currentUsername!!)
         platformRepo.updateDashPayProfile(emptyProfile)
         if (blockchainIdentityData.creationState < CreationState.DONE) {
-            platformRepo.updateCreationState(blockchainIdentityData, CreationState.DONE)
+            platformRepo.updateIdentityCreationState(blockchainIdentityData, CreationState.DONE)
+            if (wallet.balance.isGreaterThan(Constants.DASH_PAY_FEE)) {
```

Comment:
`wallet.balance` only includes confirmed transactions (confirmed on the blockchain).

if the user does not have sufficient balance during username creation gets a transaction later, that is after username creation, the alert won't appear on the Notifications Screen.  Users that are invited won't have a balance and will never see the alert.

I think there are notes that this should be handled in a different story/PR.

Reply:
@HashEngineering I asked question related to this on the Jira ticket and the answer was that there'll be another ticket to handle more complex scenarios.